styles/mixins/patterns/_molecules.scss

```diff
@@ -74,7 +74,32 @@ $-color-background: var.$color-background;
   }
 }
 
-// A responsive sized and positioned container for modal content
+/**
+ * A modal container, with responsive positioning and sizing.
+ *
+ * A modal should contain one immediate-child element; in most cases, an
+ * element that applies the dialog[1] pattern. Content within the dialog
+ * (grandchild elements of modal) can be managed with `overflow` rules so as not
```

Comment:
Could `Modal` component create this direct-child element and set the `overflow` style?

Reply:
I want to give authors flexibility over which elements scroll or don't scroll. Not all modals need scrolling content, and not all scrolling content will be the entirety of the modal content, if that makes sense. These rules merely make it _possible_ to apply `overflow`, if that makes sense. I don't want to dictate what overflows.

Reply:
Yes, it makes sense. Your approach is more flexible but it requires more from the developer.

My comment assumed that the most likely scenario is that the scrollable content will be the entirety of the modal.